test/scripts/0-to-0xff.py

```diff
@@ -0,0 +1,8 @@
+import sys
```

Comment:
i dont think we need python for this. r2 can do that already

Reply:
the script was originally for the `prc` test and it wasn't just doing `woe` but also adding flags https://github.com/radareorg/radare2/pull/16838/commits/d11f441e29a9b02f11f237037782aa04fb48dc33#diff-a5c0ff5da9addea0d27f857148c32f57 so unless r2 can do loops, it will have to stay